index.js

```diff
@@ -31,25 +31,19 @@ class Remedian {
   }
 
   /**
-     * Check if buffer is empty
-     * @param {number} position - buffer number to check.
-     * @return {boolean} true if buffer is empty, false if not.
-     */
+   * Check if buffer is empty
+   * @param {number} position - buffer number to check.
+   * @return {boolean} true if buffer is empty, false if not.
+   */
   _bufferEmpty(position) {
-    const array = this.buffers[position];
-    for (let j = 0; j < array.length; j++) {
-      if (array[j] != undefined) {
-        return false;
-      }
-    }
-    return true;
+    return this.buffers[position][0] != undefined ? false : true;
```

Comment:
```suggestion
    return this.buffers[position][0] == undefined
```

Reply:
Yeah, that was quick. Actually, this is my first Open Source PR that's why there were some issues. I have resolved the things you mentioned.

Reply:
No problem if this is your first contribution. There are still some things to fix. I will text you later today/tomorrow with some help on how to fix things.